### PR TITLE
Add sphinxcontrib bibtex

### PR DESCRIPTION
**Pull request by croth1**
:tada:

### CODE REVIEW

**Comment by croth1** [comment]
@souravsingh, @conda-forge/core - should be ready for a first round of reviews.

File: recipes/oset/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - oset
+
+about:
+  home: https://gitorious.com/sleipnir/python-oset
+  license: PSF
```
Comment (souravsingh) [inline recipes/oset/meta.yaml]: Is PSF license accepted by conda-forge? Just asking for curiosity.

**Comment by souravsingh** [comment]
Apart from the one question, Rest looks good to me. I am closing the PR I made for sphinxcontrib-bibtex.

**Comment by croth1** [comment]
@conda-forge/core I'm getting timeout errors on appveyor 👎 Can someone restart?

File: recipes/oset/meta.yaml (hunk 1 above)
Comment (croth1) [inline recipes/oset/meta.yaml]: I have seen it in a number of other recipes, so I guess it's fine. ping @conda-forge/core

**Pull request merged**